### PR TITLE
git-bars: init at 2023-08-08

### PR DESCRIPTION
**Pull request by matthiasbeyer**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-bars/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, git }:
+
+python3Packages.buildPythonApplication rec {
```
Comment (ius) [inline pkgs/applications/version-management/git-bars/default.nix:3]: Your derivation require `rec` as far as I can tell?

**Review by ius — APPROVED** [review]
Minor comment, otherwise looks fine

File: pkgs/applications/version-management/git-bars/default.nix (hunk 1 above)
Comment (matthiasbeyer) [reply pkgs/applications/version-management/git-bars/default.nix]: Fixed.

**Review by ius — APPROVED** [review]

File: pkgs/applications/version-management/git-bars/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, git }:
+
+python3Packages.buildPythonApplication {
+  pname = "git-bars";
+  version = "2023-08-08";
```
Comment (AndersonTorres) [inline pkgs/applications/version-management/git-bars/default.nix:5]:
```suggestion
  version = "unstable-2023-08-08";
```

File: pkgs/applications/version-management/git-bars/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/knadh/git-bars";
+    description = "A utility for visualising git commit activity as bars on the terminal";
+    license = licenses.unfree; # no license in repository
+    maintainers = [ maintainers.matthiasbeyer ];
+  };
+}
+
```
Comment (AndersonTorres) [inline pkgs/applications/version-management/git-bars/default.nix:26]:
```suggestion
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/version-management/git-bars/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  propagatedBuildInputs = [
+    git
+    python3Packages.setuptools
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/knadh/git-bars";
+    description = "A utility for visualising git commit activity as bars on the terminal";
+    license = licenses.unfree; # no license in repository
```
Comment (AndersonTorres) [inline pkgs/applications/version-management/git-bars/default.nix:22]:
```suggestion
    license = licenses.mit;
```

It was updated yesterday. What about update this expression too?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

**Pull request merged**